composer/callbacks/system_metrics_monitor.py

```diff
@@ -83,3 +136,17 @@ def compute_system_metrics(self):
         for k, v in network_usage.items():
             system_metrics[f'network_{k}'] = v
         return system_metrics
+
+    def compute_min_max_metrics(self, all_metrics, model_device):
+        min_max_metrics = {}
+
+        if self.gpu_available:
+            for key in _GPU_METRICS:
+                values = torch.tensor([metrics_for_cur_rank[key] for metrics_for_cur_rank in all_metrics],
+                                      device=model_device)
+                min_rank = int(torch.argmin(values).item())
+                max_rank = int(torch.argmax(values).item())
+                min_max_metrics[f'min_{key}/Rank_{min_rank}'] = values[min_rank].item()
+                min_max_metrics[f'max_{key}/Rank_{max_rank}'] = values[max_rank].item()
```

Comment:
```suggestion
                min_max_metrics[f'min_{key}_rank_{min_rank}'] = values[min_rank].item()
                min_max_metrics[f'max_{key}_rank_{max_rank}'] = values[max_rank].item()
```
to be consistent with other log